src/main/scala/akka/persistence/cassandra/journal/CassandraStatements.scala

```diff
@@ -19,8 +19,10 @@ trait CassandraStatements {
         persistence_id text,
         partition_nr bigint,
         sequence_nr bigint,
+        timestamp timeuuid,
+        tag text,
         message blob,
-        PRIMARY KEY ((persistence_id, partition_nr), sequence_nr))
+        PRIMARY KEY ((persistence_id, partition_nr), sequence_nr, tag))
```

Comment:
I did not have a chance to verify, but my expectation was to be able to have any number of tags per message. That means we need to duplicate the messages in cassandra (because collections are not an option for MVs). It complicates things such as replay, but seems to be conceptually the right thing to do.